Library/Formula/libfixposix.rb

```diff
@@ -0,0 +1,46 @@
+class Libfixposix < Formula
+  desc "Thin wrapper over POSIX syscalls"
+  homepage "https://github.com/sionescu/libfixposix"
+  url "https://github.com/sionescu/libfixposix/archive/v0.3.0.tar.gz"
+  sha256 "9fda592842c28b3beb2707b908b2bdca1982c0a37572fad5cfce2ab1ba07f6d3"
+  head "https://github.com/sionescu/libfixposix.git"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "autoreconf", "-i", "-f"
```

Comment:
You can happily combine `autoreconf` arguments in the latter part here, i.e. `"-fvi"`. Matches what we use elsewhere. Also has the side-effect of triggering a new bottle build since the last one didn't start for some reason :smile_cat: